pxtcompiler/emitter/backjs.ts

```diff
@@ -554,7 +554,7 @@ function ${id}(s) {
                     write(`if ((${args[0]}) && (${args[0]}).vtable) {`)
                 }
                 if (topExpr.callingConvention == ir.CallingConvention.Promise) {
-                    write(`(function(cb) { ${text}.done(cb) })(buildResume(s, ${loc}));`)
+                    write(`(function(cb) { ${text}.then(cb) })(buildResume(s, ${loc}));`)
```

Comment:
>then [](start = 51, length = 4)

The code being emitted here.. was the environment where it executes also upgraded from bluebird?

Reply:
Right, all code in the frontend currently uses bluebird (`.done` isn't actually part of the ecmascript promise spec in the first place)